vowpalwabbit/slim/CMakeLists.txt

```diff
@@ -1,8 +1,3 @@
-cmake_minimum_required(VERSION 3.5)
```

Comment:
why are these not needed anymore

Reply:
You cannot actually build the slim project "standalone" as it depends on the explore target. Therefore having these here is not useful